ui/perfherder/compare/CompareSubtestDistributionView.jsx

```diff
@@ -122,6 +126,12 @@ export default class CompareSubtestDistributionView extends React.Component {
       }),
     );
 
+    if (seriesData.failureStatus) {
+      return this.setState({
+        failureMessages: [...seriesData.data, ...failureMessages],
```

Comment:
If this is the only place we're updating the `failureMessage` state then you only need to do: `failuresMessages: seriesData.data` (check if it returns as an array or string).

Reply:
PR coming by tomorrow, sorry for the delay